Dockerfile

```diff
@@ -334,56 +380,52 @@ COPY --chown=airflow:airflow airflow/version.py ${AIRFLOW_SOURCES}/airflow/versi
 COPY --chown=airflow:airflow airflow/__init__.py ${AIRFLOW_SOURCES}/airflow/__init__.py
 COPY --chown=airflow:airflow airflow/bin/airflow ${AIRFLOW_SOURCES}/airflow/bin/airflow
 
-# The goal of this line is to install the dependencies from the most current setup.py from sources
-# This will be usually incremental small set of packages in CI optimized build, so it will be very fast
-# In non-CI optimized build this will install all dependencies before installing sources.
-RUN pip install -e ".[${AIRFLOW_EXTRAS}]"
-
+# Setting to 1 speeds up building the image. Cassandra driver without CYTHON saves around 10 minutes
+# But might not be suitable for production image
+ENV CASS_DRIVER_NO_CYTHON=""
+ENV CASS_DRIVER_BUILD_CONCURRENCY="8"
 
-WORKDIR ${AIRFLOW_SOURCES}/airflow/www
-
-# Copy all www files here so that we can run npm building for production
-COPY --chown=airflow:airflow airflow/www/ ${AIRFLOW_SOURCES}/airflow/www/
+ENV PATH="/home/airflow/.local/bin:/home/airflow:${PATH}"
 
-# Package NPM for production
-RUN gosu ${AIRFLOW_USER} npm run prod
+# The goal of this line is to install the dependencies from the most current setup.py from sources
+# This will be usually incremental small set of packages in CI optimized build, so it will be very fast
+# For production optimised build it is the first time dependencies are installed so it will be slower
+RUN pip install --user ".[${AIRFLOW_PROD_EXTRAS}]" \
+    && pip uninstall --yes apache-airflow snakebite
```

Comment:
Snakebite is removed - i did not want to remove it from deps as it is currently installed and some of the pylint/imports wont work correctly for those packages using snakebite (which is basically all hadoop operators). I will do it differently- modifying  setup.py and see if that works (I think I had some problems when removed snakebite)

Reply:
I think I remember why I've done it. For the CI build we currently have snakebite as part of master when installed few lines above. I had to remove it anyway at some point in time. But it will not be needed here if we remove it from setup.py

Reply:
Yeah. We need it for now.